cmd/inspect.go

```diff
@@ -77,21 +83,32 @@ func init() {
 
 	inspectCmd.Flags().BoolVarP(&details, "details", "", false, "print all details of lab containers")
 	inspectCmd.Flags().StringVarP(&format, "format", "f", "table", "output format. One of [table, json]")
+	inspectCmd.Flags().BoolVarP(&all, "all", "a", false, "show all containerlab labs in the host")
```

Comment:
"show all running containerlab labs"?

"on the host" seems to be irrelevant, since we do not support multi host deployments. at the same time adding "running" makes sense to me, since that excludes assumption that this command will analyze labs which are not running, but defined in the files in the CWD

Reply:
apart from that the rest lgtm and it works as expected

Reply:
changed